as2/server/mail.md

```diff
@@ -0,0 +1,89 @@
+# Mail
+##### Messages that pertain to users' mail system.
+---
+### Delete Mail From Player
+Sent to clients when a postcard has been deleted from their penguin's mail.
```

Comment:
Maybe add a note that this is usually only done at the user's request.

Reply:
Just as well, this particular message is for removing __all__ postcards that are from a specific penguin and this should be noted.

This message also cannot be sent arbitrarily as it requires a client request - otherwise it will be ignored.